packages/editor/src/lib/editor/Editor.ts

```diff
@@ -6582,7 +6587,11 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 *
 	 * @public
 	 */
-	groupShapes(shapes: TLShapeId[] | TLShape[], groupId = createShapeId()): this {
+	groupShapes(
+		shapes: TLShapeId[] | TLShape[],
+		groupId = createShapeId(),
+		options: TLGroupShapesOptions = { selectAfterGrouping: true }
```

Comment:
I think it would be better to combine these into one options argument while preserving backwards compatibility. something like this

```ts
export type TLGroupShapesOptions = {
	selectAfterGrouping?: boolean
  shapeId?: string
}

// ...


groupShapes(shapes: TLShapeId[] | TLShape[], options: TLGroupShapesOptions | TLShapeId = {}) {
  if (typeof options === 'string') {
    options = { groupId: options }
  }

  // ...
}
```
That way we keep the 'options' a bit tidier.